src/NServiceBus.MessagingBridge/NServiceBus.MessagingBridge.csproj

```
@@ -13,8 +13,11 @@
 
   <ItemGroup>
     <PackageReference Include="Particular.Packaging" Version="4.1.0" PrivateAssets="All" />
+    <PackageReference Include="Fody" Version="6.8.1" PrivateAssets="All" />
+    <PackageReference Include="Janitor.Fody" Version="1.9.0" PrivateAssets="All" />
+    <PackageReference Include="Obsolete.Fody" Version="5.3.0" PrivateAssets="All" />
```

Janitor not needed for obsoletes

```suggestion
    <PackageReference Include="Fody" Version="6.8.1" PrivateAssets="All" />
    <PackageReference Include="Obsolete.Fody" Version="5.3.0" PrivateAssets="All" />
```